src/main/java/io/supertokens/oauth/OAuth.java

```diff
@@ -386,4 +467,41 @@ public static JsonObject introspectAccessToken(Main main, AppIdentifier appIdent
         result.addProperty("active", false);
         return result;
     }
+
+    public static void revokeAllConsentSessions(Main main, AppIdentifier appIdentifier, Storage storage, String subject, String clientId) throws StorageQueryException {
+        OAuthStorage oauthStorage = StorageUtils.getOAuthStorage(storage);
+        if (clientId == null) {
+            oauthStorage.revoke(appIdentifier, "sub", subject);
+        } else {
+            oauthStorage.revoke(appIdentifier, "client_id_sub", clientId + ":" + subject);
```

Comment:
What's the usecase for `clientId` + `sub`? I think `client_id` would make sense on it's own though to rotate all credentials in case the `clientSecret` was leaked.

Reply:
I was proxying the wrong endpoint (got misled by the link specified in the check list). fixed this now.